plugin-animation/src/test/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImplTest.kt

```diff
@@ -962,6 +963,21 @@ class CameraAnimationsPluginImplTest {
     assertEquals(10.0, updateList[1].pitch!!, EPS)
   }
 
+  @Test
+  fun debugModeTest() {
+    // debug mode is enabled by default for backward compatibility
+    shadowOf(getMainLooper()).pause()
+    cameraAnimationsPluginImpl.easeTo(cameraOptions, mapAnimationOptions { duration(DURATION) })
+    shadowOf(getMainLooper()).idle()
+    assertNotEquals(0, ShadowLogger.logCount)
```

Comment:
If I understand correctly when debug mode is enabled the logcount should not be 0.

Reply:
Correct, that's why I'm using here `assertNotEquals` 😉